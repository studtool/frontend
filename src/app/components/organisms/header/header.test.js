import React from 'react';
import Header from './header';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import {BrowserRouter} from 'react-router-dom';


test('Header test', () => {
    const tree = renderer.create(
        <BrowserRouter>
            <Header>
                test
            </Header>
        </BrowserRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
