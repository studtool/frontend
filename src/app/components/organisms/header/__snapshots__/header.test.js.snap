// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test 1`] = `
<div
  className="header"
>
  <h1>
    Header
  </h1>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      <button>
        Home
      </button>
    </a>
  </div>
</div>
`;
